web/src/actions/app/navigation.js

```diff
@@ -1,28 +1,8 @@
 import { push } from 'connected-react-router';
-import { NAVIGATION_SCROLL_TO_WAYPOINT, RESET } from './symbols';
-
-let jumping = false;
+import { RESET } from './symbols';
 
+// eslint-disable-next-line import/prefer-default-export
 export const goToDashboard = ({ pushRoute = push } = {}) => dispatch => {
   dispatch({ type: RESET });
   dispatch(pushRoute('/'));
 };
-
-export const jumpTo = waypointId => dispatch => {
-  if (waypointId) {
-    jumping = true;
-    setTimeout(() => {
-      jumping = false;
-    }, 100);
-    dispatch({ type: NAVIGATION_SCROLL_TO_WAYPOINT, waypointId });
-  }
-};
-
-export const scrollTo = waypointId => dispatch => {
-  if (!jumping && waypointId) {
-    dispatch({
-      type: NAVIGATION_SCROLL_TO_WAYPOINT,
-      waypointId
-    });
-  }
-};
```

Comment:
`jumpTo` and `scrollTo` actions go away. This functionality is handled by `react-router`, `connected-react-router`, the `<AutoScrollToElement/>` component, and the `<ConnectedWaypoint/>` component.